src/packages/cli_manager/cli_manager.py

```diff
@@ -2,7 +2,7 @@
 cli_manager.py - Manage CLI arguments
 
 This module will automatically parse arguments provided from the command line when this module is
-**first imported** and provide it for importing by other modules via the `args` attribute
+**first imported** and provide it for importing by other modules via the `GET_ARGUMENTS` attribute
```

Comment:
[Error] `GET_ARGUMENTS` is not an attribute, it is a callable.

Reply:
not strictly true. it is an attribute as it is a member of a (module) object.

That said, the more apt description of this object is now `method` and will be adjusted

Reply:
Resolved in last push.